docker-compose-services/mysql/README.md

```diff
@@ -0,0 +1,40 @@
+For the vast majority of use cases where MySQL is needed in a project, MariaDB is the best choice. MariaDB is faster and more feature-packed in many scenarios. However, sometimes a project really demands plain old MySQL.
+This page documents how to create a separate MySQL container to use with ddev. It does not
+replace the MariaDB or PHPMyAdmin containers. It also demonstrates how to connect Drupal to that container and database.
+
+## ddev setup
+1. Add [docker-compose.mysql.yaml](docker-compose.mysql.yaml) to your  project's `.ddev` directory.
+2. Start or restart your project with `ddev start` or `ddev restart`.
+3. You should see your container start up. You could look for it with `docker ps -a | grep mysql`. The container will be named ddev-[your project name]-mysql.
+
+NB: We use MySQL 5.7, but it should be possible to use a different version. Check out the [MySQL Images on Docker Hub](https://hub.docker.com/_/mysql) for more options. Then change the image key in docker-compose.mysql.yaml.
+
+## Connecting
+If your container was successfully started, you should be able to access it from within the ddev webserver container.
```

Comment:
This section would be better if you ssh into the mysql container. `ddev ssh -s mysql`. The other way doesn't work at all with mysql 8. You also then don't have to add the --database=mysql in step 2 because that's the default.